share/blob.go

```diff
@@ -149,3 +150,20 @@ func SortBlobs(blobs []*Blob) {
 		return blobs[i].Compare(blobs[j]) < 0
 	})
 }
+
+// IsValidBlobNamespace returns a true if this namespace is a valid user-specifiable
+// blob namespace.
+func IsValidBlobNamespace(ns Namespace) bool {
```

Comment:
Seems like we have two patterns for namespace validators func signatures.
* ValidateForData
* IsValidBlobNamespace.

Worth unifying them.

[blocking]

Reply:
Yeah `ValidateForData` seems redundant

Reply:
I think @Wondertan meant semantical unification.